tests/fixtures/fixtures_auth.py

```diff
@@ -43,10 +43,20 @@ def auth_mock(requests_mock):
 
 
 @pytest.fixture(scope="module")
-def auth_live():
+def project_id_live():
+    return os.getenv("TEST_UP42_PROJECT_ID")
+
+
+@pytest.fixture(scope="module")
+def project_api_key_live():
```

Comment:
out of curiosity - is there any advantage of defining and passing these values as fixtures?

Reply:
To reduce the usage of auth.project_id which is going be deprecated and isolate the environment dependencies as explicit fixtures